README.md

```diff
@@ -20,68 +20,66 @@ Like the TSC governance, all issues and pull requests follow a Consensus Seeking
+Before any Pull Request is landed, sufficient time should be given to receive input from other Collaborators. Specifically, at least 48 hours during the typical working week and 72 hours over weekends should be given to account for international time differences and work schedules. Trivial Pull Requests may be landed after a shorter delay.
 
-Specific Collaborators can be requested to review any PR by using an *@-mention* either within the PR text itself or the associated comment stream.
+If any Collaborator feels that a particular PR is controversial and needs to be reviewed further, the Collaborator can call attention to the Pull Request by attaching either the *tsc-agenda* tag (to put the Pull Request on the TSC agenda) or a Working Group specific tag to indicate that the PR requires review from a specific Working Group. PR’s that are put onto the TSC or WG agenda cannot be landed until the tagged WG or TSC has the opportunity to review and decide using the Consensus Seeking Model.
 
-Collaborators sign off on a PR by explicitly stating their approval in the PR text or associated comment stream. If a Collaborator is unsure about the modification or is not prepared to take full responsibility for the change, they should defer to another Collaborator. 
+Elevationg of Pull Requests or issues to the TSC or WG should occur when the Pull Request:
 
-Before landing a Pull Request, sufficient time should be given to receive input from other Collaborators. Specifically, at least 48 hours during the typical working week and 72 hours over weekends should be given to account for international time differences and work schedules. Trivial changes (e.g. those which fix minor bugs or improve performance without affecting API or causing other wide-reaching impact) may be landed after a shorter delay. 
+* has a significant impact on the codebase,
+* is inherently controversial; or
+* has failed to reach consensus amongst the Collaborators who are actively participating in the discussion.
 
-For high priority changes that address security vulnerabilities, a shorter review period may be necessary depending on the nature of the change and severity of the issue. It is recommended that the TSC establish a special Security Working Group of Collaborators with recognized security expertise who can be tasked with reviewing security related Pull Requests.
+The TSC serves as the final arbiter where required but it is expected that most changes reaching a lazy consensus are merged without TSC intervention.
```

Comment:
The original text is grammatically correct, if a bit of a mind-bender: "reaching a lazy consensus" is a participle phrase functioning as an adjective describing "changes", which is further quantified by "most". Thus "The TSC serves as the final arbiter where require but it is expected that X's are merged without TSC intervention", where "X's" -> "most Y's'", "Y's" -> "changes reaching a lazy consensus". The substitution makes the sentence grammatically incorrect (it adds two verbs to a single clause – "will reach" and "are").

**That said**, this sentence might need some unpacking – "The TSC serves as the final arbiter where required. However, it is expected that most changes will reach a lazy consensus. Those changes may be merged without TSC intervention."